pyproject.toml

```diff
@@ -13,5 +13,9 @@ extend-exclude = '''
 )/
 '''
 
+[tool.ruff]
+line-length = 120
+target-version = "py38"  # https://github.com/rwth-i6/returnn/issues/1326
```

Comment:
I don't exactly understand. Is this not used? Why do you have reformatted some files with Python 3.8 incompatible changes?

Reply:
First I did not have this line there, saying the results must be 3.8 compatible. But I formatted. Then I added the line and formatted again. ruff did not re-format the constructs that are invalid in 3.8, though. I did a clean checkout, added the 3.8 config and then did a format, and it outputted 3.8-compatible constructs.

Reply:
So can you update this PR then with the correct formatting change?

Reply:
It is correct now.